pkg/dao/repository_configs.go

```diff
@@ -387,7 +387,8 @@ func (r repositoryConfigDaoImpl) filteredDbForList(OrgID string, filteredDB *gor
 		filteredDB = filteredDB.Where("name = ?", filterData.Name)
 	}
 
-	if !config.Get().Features.NewRepositoryFiltering.Enabled && filterData.ContentType == "" {
+	// Default type to ContentTypeRpm if not specified
+	if filterData.ContentType == "" {
 		filterData.ContentType = config.ContentTypeRpm
```

Comment:
I think we want to not default to RPM if its disabled.  The point of 'newrepositoryfiltering' was that there wasn't any filtering and a fetch of  /repositories/  returns everything